game/src/main/scala/io/aigar/game/Game.scala

```diff
@@ -1,32 +1,47 @@
 package io.aigar.game
 
 import io.aigar.game._
+import com.github.jpbetz.subspace._
 
 /**
  * Game holds the logic for an individual game being played
  * (e.g. the ranked game or a private test game).
  */
 object Game {
   final val RankedGameId = 0
+  final val PlayersInRankedGame = 15
```

Comment:
We put that in a constant? So if we want to change that we need to close the server? Is there a use-case where we would need to change the number of players?